src/weapons.cpp

```diff
@@ -772,7 +781,7 @@ bool WeaponDistance::useWeapon(Player* player, Item* item, Creature* target) con
 	}
 
 	if (item->getWeaponType() == WEAPON_AMMO) {
-		Item* bow = player->getWeapon(true);
+		auto bow = player->getWeapon(true);
```

Comment:
Could this be `const auto`
```suggestion
		cont auto bow = player->getWeapon(true);
```